src/kern_db.cpp

```diff
@@ -57,7 +57,7 @@ KernDb::KernDb(const std::string& filename_,
         const auto lock = exclusive_lock(lock_file, GetLockTimeout());
         MIOPEN_VALIDATE_LOCK(lock);
         const std::string create_table = KernelConfig::CreateQuery();
-        if(!SQLExec(create_table))
+        if(!sql.Exec(create_table))
```

Comment:
There is no point in checking the return type as it will always return true or throw on error.

Reply:
fixed